packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx

```diff
@@ -29,9 +29,9 @@ import {
   IRefArrayData,
   Legends,
 } from '../../index';
+import { formatPrefix as d3FormatPrefix } from 'd3-format';
```

Comment:
The gap within a group is looking less for the basic example.

Reply:
increased padding